spec/controllers/students_controller_spec.rb

```diff
@@ -60,4 +60,17 @@
         end
     end  
   end
+
+  describe 'DELETE #destroy' do
+    subject { delete :destroy, id: student }
+
+    it 'redirects to the student list upon deletion' do
+      is_expected.to redirect_to students_path
+    end
+
+    it 'deletes the expected student' do
+      subject
+      expect { Student.find(student.id) }.to raise_error(ActiveRecord::RecordNotFound)
```

Comment:
You can use the `.reload` method to let activerecord reload the student from the database, (ie. `student.reload`). You can also use the `be` matcher to check that the record still exists. Combining the two, we end up with something like 

 ```suggestion
      subject
      expect(student.reload).to_not be
```

which IMO reads a bit nicer, what do you think ?

Reply:
It does, although `student.reload` also seems to raise `RecordNotFound`, so instead I have replaced it the following:
```rb
expect { student.reload }.to raise_error(ActiveRecord::RecordNotFound)
```

Reply:
Yep that'll do 👍